fetch-in-react/my-app/src/User.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import UserCard from './UserCard';
+
+export default function User({ userId, onCancel }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [user, setUser] = useState();
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+    fetch('https://jsonplaceholder.typicode.com/users/' + userId)
```

Comment:
Please use async/away syntax instead of Promise syntax.

Otherwise, it's OK, though.

Reply:
Oh okay, I thought that was just for the prev exercise. Will use async/await syntax exclusively moving forward. Thanks!

Reply:
Fixed!